### DISCUSSION TITLE
Questions for CyberGlow Theme

### DISCUSSION THREAD

**Discussion by ThePharaohArt** [ArtexJay/Obsidian-CyberGlow#82] [Q&A]
If you have any questions about my theme, the pictures used in the preview or anything in relation to my theme, please ask them here in this thread.

I will do my best to reply as soon as I can..

**Comment by mechaji** [comment]
Hello! I'm new to Obsidian and I'm using your wonderful CyberGlow theme. I was wondering if there is a way to hide the H1, H2 titles next to the different sized headings? Thank you!

**Comment by ThePharaohArt** [comment]
> Hello! I'm new to Obsidian and I'm using your wonderful CyberGlow theme. I was wondering if there is a way to hide the H1, H2 titles next to the different sized headings? Thank you!

Not at this time, I will add the option to disable it though.
Edit: I have the option working and will be released in the next update. I'm planning for it to come out tomorrow.

**Comment by mechaji** [comment]
Thank you so much!

**Comment by ThePharaohArt** [comment]
> Thank you so much!

The Rewrite Update is out now, you should be able to see the setting you want in Style Settings under Heading Controls, called Hide Heading Indication.
![image](https://github.com/ArtexJay/Obsidian-CyberGlow/assets/32932497/e9dcf186-8e94-4b38-9d3a-55d9686c7a50)

**Comment by ThePharaohArt** [comment]
A few people have asked me what was the image I used on the background here
![image](https://github.com/ArtexJay/Obsidian-CyberGlow/assets/32932497/07d3710c-66f4-4c39-bf45-859b20e63c12)

It was this Pokemon Go art by Sakimichan (They make crazy good art) https://www.deviantart.com/sakimichan/art/Go-623780055

**Comment by CamoCatX** [comment]
How difficult would this be to convert to another note taking app, such as logseq?

**Reply by ThePharaohArt** [reply]
I am open to doing this

**Reply by ThePharaohArt** [reply]
Looked at Logseq real quick, I believe this should be easy to do

**Comment by CyRamos** [comment]
Hey :) 
I just installed your theme and it looks fantastic. 
I'd like to know if there are built-in functions such ITS-Theme for adjustments in-notes like: 
- [Captions](https://publish.obsidian.md/slrvb-docs/ITS+Theme/Callouts/Callout+-+Captions) 
- [Grid](https://publish.obsidian.md/slrvb-docs/ITS+Theme/Callouts/Callout+-+Grid) 
- [Infoboxes ](https://publish.obsidian.md/slrvb-docs/ITS+Theme/Callouts/Callout+-+Infoboxes)

Thanks again for your effort, it looks amazing.

**Reply by ThePharaohArt** [reply]
not at this time, though I do want to implement those
I am currently going through the code and doing a massive update with tons of changes, fixes and new additions.

**Reply by ThePharaohArt** [reply]
I have managed to implement Infoboxes, looking into grid and captions

**Reply by ThePharaohArt** [reply]
@CyRamos  Just released the newest update infoboxes,  grid and captions are supported now

**Comment by WickedSquirrel** [comment]
Is there a way to remove the icons from the list of items and folders?
I have added my own with iconizer and they overlap.

**Reply by ThePharaohArt** [reply]
not at this time, I believe I should be able to do this, never used iconizer myself so Ill check it out

**Reply by ThePharaohArt** [reply]
You will be able to disable it in the next update that I will release in the coming days

**Reply by ThePharaohArt** [reply]
@Daemonias Just released the newest update, the setting can be found in Style Settings.

**Comment by WickedSquirrel** [comment]
Is there a way to change the colour in bulk?
right now I'd have to click a hundred times to change the colour of everything and see how it feels

**Reply by ThePharaohArt** [reply]
which elements do you want to change the color of?
Have any suggestions which elements you think would be best grouped?

**Reply by WickedSquirrel** [reply]
hard to say
basically all that have the same colour? 
Not sure how other Themes decide on that.. but there seems to be a lot of precedent

**Comment by violet-nova** [comment]
Hi, do you know of a way to publish to GitHub Pages while preserving this theme? The Obsidian Quartz Template sadly does not do it, at least not out of the box.
